### PR TITLE
Corpus

### PR DESCRIPTION
**Pull request by samouri**
- Updated the `README.md` with descriptions on how the corpus was generated.
- Added `manifest.json` with links to original articles
- Removed wapo since everything is behind a paywall or outdated
- Added 2 ecommerce and 2 newsy sites.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+// For SSR.
+document.body.querySelectorAll('iframe').forEach(node => {
+  node.setAttribute('loading', 'lazy')
+})
+copy(document.documentElement.innerHTML)
+// --> manually replace amp-carousel --> amp-carousel2 so it doesn't hide itself immediately during build.
+
+// For nojs
+document.querySelectorAll('script[src]').forEach(node => node.remove())
+document.querySelectorAll('script[custom-element]').forEach(node => node.remove())
```
Comment (rcebulko) [inline README.md:23]: nit, please ignore unless you're already making other changes to this file

```javascript
document.querySelectorAll('script[src],script[custom-element]').forEach(node => node.remove())
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
 # amp-performance-experiments
 
 1. gather corpus
+  1.1 use view-source and copy/paste to generate the `index.default.html`
+  1.2 use Chrome and load the same page. Modify all iframes to have loading=lazy. `copy(document.innerHTML)` to generate `index.ssr.html`. Rename significant amp-carousel components to amp-carousel2.
+  1.3 remove all extension script tags from the `ssr.html` file to generate `index.nojs.html`
```
Comment (rcebulko) [inline README.md:6]: Are there any steps you'd recommend to validate that doing this didn't break anything, other than a visual lookover?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+
+
+## Appendix
+
+```javascript
+// For SSR.
+document.body.querySelectorAll('iframe').forEach(node => {
+  node.setAttribute('loading', 'lazy')
+})
+copy(document.documentElement.innerHTML)
```
Comment (rcebulko) [inline README.md:18]: Good idea adding these scripts!

File: corpus/manifest.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "ampdev": "https://amp.dev",
+    "bedbath": "https://www-bedbathandbeyond-com.cdn.ampproject.org/c/s/www.bedbathandbeyond.com/amp/store/product/calphalon-reg-premier-stainless-steel-11-piece-cookware-set/5504632",
+    "bh": "https://www-bhphotovideo-com.cdn.ampproject.org/c/s/www.bhphotovideo.com/explora/amp/pro-audio/hands-on-review/teenage-engineering-od-11-wireless-cloud-speaker",
+    "foxnews": "https://www-foxnews-com.cdn.ampproject.org/c/s/www.foxnews.com/us/us-marshals-increase-reward-to-10000-for-person-of-interest-in-yale-student-kevin-jiangs-murder.amp",
+    "msnbc": "https://www-msnbc-com.cdn.ampproject.org/c/s/www.msnbc.com/msnbc/amp-video/mmvo101120581661",
+    "pinterest": "https://www-pinterest-com.cdn.ampproject.org/c/s/www.pinterest.com/amp/bestbullysticks/puppies"
+}
```
Comment (rcebulko) [inline corpus/manifest.json:8]:
```suggestion
}

```

Comment (rcebulko) [reply corpus/manifest.json:8]: Is our standard here that each key maps to a directory, ie. ``Object.keys(manifest).map(name => `./${name}/index.${variant}.html`)`` is the set of test files?

**Review by rcebulko — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (samouri) [reply README.md:6]: I've been visually looking over everything. which is actually how i learned that amp-carousel needs to be tweaked

File: README.md (hunk 1 above)
Comment (samouri) [reply README.md:23]: you can comma separate selectors!?!?!? TIL

File: corpus/manifest.json (hunk 4 above)
Comment (samouri) [reply corpus/manifest.json:8]: Yes. At least I'm trying to do that, although it isn't enforced anywhere.
I'd like to think that in some ideal world where I had infinite time the process of gathering the corpus could be fully derived from the manifest and a pupeteer-controlled browser

**Pull request merged**